src/main/java/mekanism/common/entity/AllayRobitSkin.java

```diff
@@ -0,0 +1,28 @@
+package mekanism.common.entity;
+
+import mekanism.api.robit.RobitSkin;
+import mekanism.common.Mekanism;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class AllayRobitSkin extends RobitSkin {
+
+    public AllayRobitSkin() {
+        super(new ResourceLocation[]{
+                Mekanism.rl( "allay"), Mekanism.rl("allay2")}
+        );
+    }
+
+    @Nullable
+    @Override
+    public ResourceLocation getCustomModel() {
+        return Mekanism.rl("item/robit_allay");
```

Comment:
`private static final ResourceLocation MODEL =` (This method gets called a fair bit iirc)

Reply:
just once for each skin and once per frame for each rendered item, but will change that